chips/stm32f3xx/src/deferred_call_tasks.rs

```diff
@@ -0,0 +1,30 @@
+//! Definition of Deferred Call tasks.
+//!
+//! Deferred calls allow peripheral drivers to register pseudo interrupts.
+//! These are the definitions of which deferred calls this chip needs.
+
+use core::convert::Into;
+use core::convert::TryFrom;
+
+/// A type of task to defer a call for
+#[derive(Copy, Clone)]
+pub enum Task {
+    Nop = 0,
```

Comment:
What is a `Nop` task?

Reply:
I based my implementation on the stm32f4xx, this was there. Will look into it, not sure at what it was used.

Reply:
I think this is just an example of a task, should I remove it?